test/print.jl

```diff
@@ -631,12 +632,15 @@ end
         io_test(REPLMode, model, """
     Max sin(x)
     Subject to
-     cos(x) - 0.0 $eq 0
+     subexpression[1] - 0.0 $eq 0
+    With NL expressions
+     subexpression[1]: cos(x)
```

Comment:
Why `:` and not ` =` ?

Reply:
I considered that but I thought it could be confused with equality constraints, especially on a large model where "With NL expressions" may not be visible on the screen.

Reply:
That makes sense